BeerOverflowWindowsApp/Form1.cs

```diff
@@ -1,57 +0,0 @@
-﻿using Newtonsoft.Json;
-using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Drawing;
-using System.Linq;
-using System.Net.Http;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Forms;
-using static TestProject.DataModels.GeodataDataModel;
-
-namespace BeerOverflowWindowsApp
-{
-    public partial class Form1 : Form
-    {
-        public Form1()
-        {
-            InitializeComponent();
-            latitudeBox.Text = "54.684815";
-            longitudeBox.Text = "25.288464";
-            radiusTextBox.Text = "500";
-        }
-
-        private void Go_Click(object sender, EventArgs e)
-        {
-            try
-            {
-                GetBarDataAsync(latitudeBox.Text, longitudeBox.Text, radiusTextBox.Text);
-            }
-            catch
-            {
-                MessageBox.Show("Something went wrong");
-            }
-        }
-        private async Task GetBarDataAsync(string latitude, string longitude, string radius)
```

Comment:
Should make a new interface with a method 'private async Task GetBarDataAsync(string latitude, string longitude, string radius)' and classes that implement it but use different API

Reply:
Implemented the interface. Please take a look. Honestly not the biggest idea what I was doing. Thanks.